cmd/aigw/ai-gateway-local.yaml

```diff
@@ -75,9 +78,10 @@ metadata:
   namespace: default
 spec:
   endpoints:
+    # Use fqdn, not ip, to allow changing to host.docker.internal
     - fqdn:
-        hostname: host.docker.internal
-        port: 11434
+        hostname: ${OPENAI_HOST:=fbi.com}  # Resolves to 127.0.0.1
```

Comment:
OLLAMA?

Reply:
added a comment at the top of the file that ramalama is port 8080